src/golang/lib/lambda/utils.go

```diff
@@ -254,9 +254,10 @@ func DeleteDockerImage(versionedLambdaImageUri string) error {
 	cmd.Stderr = stderr
 
 	err := cmd.Run()
-
 	if err != nil {
-		return errors.Wrap(err, "Unable to delete docker image.")
+		log.Info(stdout.String())
+		log.Info(stderr.String())
+		return errors.Wrap(err, "Unable to delete Docker image.")
```

Comment:
@Fanjia-Yan  Instead of logging the errors, could we include them in the `errors.Wrap` below? Sth like:
`return errors.Wrap(err, "Unable to delete Docker image. Stdout: %s. Stderr: %s.")`